packages/core/src/components/essentials/title-meter.ts

```diff
@@ -18,16 +18,17 @@ export class MeterTitle extends Title {
 		const options = this.getOptions()
 		const svg = this.getComponentContainer()
 		const { groupMapsTo } = options.data
-
+		const meterTitle = getProperty(options, 'locale', 'translations', 'meterTitle')
```

Comment:
let's try to organize these better (e.g. `options.locale.translations.meter.title`